apps/storybook-react/stories/Text.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
This is a temporary example story to easily review and test the tailwind-preset updates for typography